graph/client.py

```diff
@@ -34,8 +52,16 @@ def get_questions(self, separator):
         }]
         for rule in rules:
             questions[0]['choices'].append(dict(name=rule['id_rule']))
+        if self.show_not_selected_rules:
+            questions[0]['choices'].append(separator1)
```

Comment:
I'm confused by these separators. Why are they passed as arguments?

Reply:
It is for better testing. I'll rename them. Separator1 doesn't make much sense.